tests/tickets/9780/manual/ie10readonly.html

```diff
@@ -0,0 +1,11 @@
+<div id="editor"></div>
```

Comment:
I would also update this test name to `1.html` and `1.md` as mentioned in another comment.